R/linear_pool.R

```diff
@@ -1,14 +1,18 @@
 #' Compute ensemble model outputs as a linear pool, otherwise known as a
 #' distributional mixture, of component model outputs for
 #' each combination of model task, output type, and output type id. Supported
-#' output types include `mean`, `quantile`, `cdf`, and `pmf`.
+#' output types include `mean`, `quantile`, `cdf`, `pmf`, and `sample`.
 #'
 #' @inheritParams simple_ensemble
 #' @param n_samples `numeric` that specifies the number of samples to use when
 #'   calculating quantiles from an estimated quantile function. Defaults to `1e4`.
 #' @param ... parameters that are passed to `distfromq::make_q_fn`, specifying
 #'   details of how to estimate a quantile function from provided quantile levels
 #'   and quantile values for `output_type` `"quantile"`.
+#' @param n_output_samples `numeric` that specifies how many sample forecasts to
+#'   return per unique combination of task IDs. Defaults to NULL, in which case
```

Comment:
```suggestion
#'   return per unique combination of task IDs. Currently the only supported value is NULL, in which case
```

Reply:
I waffled a bit about whether we should just take this argument out for now.  I decided that it's ok to just leave it in because we will want/need to return to add in this piece of functionality very soon.